canvas_modules/common-canvas/src/common-properties/panels/sub-panel/cell.jsx

```diff
@@ -67,26 +67,22 @@ export default class SubPanelCell extends React.Component {
 				rejectLabel={rejectLabel}
 				controller={this.props.controller}
 			>
-				<div>
-					<Tooltip
-						id={uuid4() + "-" + tooltipId}
-						tip={subPanelToolTip}
-						direction="left"
-						className="properties-tooltips icon-tooltip"
-					>
-						<Button
-							className="properties-subpanel-button"
-							type="button"
-							size="small"
-							kind="secondary"
-							onClick={this.showSubPanel}
-							disabled={disabled}
-							aria-label={subPanelToolTip}
-						>
-							{innerObject}
-						</Button>
-					</Tooltip>
-				</div>
+				<Tooltip
```

Comment:
This is updating the edit button in the table row.  This uses a standard carbon icon button now.